demos/twa-post-message/src/main/java/com/google/androidbrowserhelper/demos/twapostmessage/MessageNotificationHandler.java

```diff
@@ -0,0 +1,50 @@
+package com.google.androidbrowserhelper.demos.twapostmessage;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import androidx.core.app.NotificationCompat;
+import androidx.core.app.NotificationManagerCompat;
+
+public class MessageNotificationHandler {
+
+  private final static String CHANNEL_ID = "TWA-postmessage-channel-id";
+
+
+  public static void showNotificationWithMessage(Context context, String message) {
```

Comment:
You need to update your IDE to use 4 spaces for indentation instead of 2 then reformat all the java code here.